packages/core/src/transport/WsOutboundTransport.ts

```diff
@@ -35,10 +35,13 @@ export class WsOutboundTransport implements OutboundTransport {
 
   public async stop() {
     this.logger.debug('Stopping WS outbound transport')
-    this.transportTable.forEach((socket) => {
-      socket.removeEventListener('message', this.handleMessageEvent)
-      socket.close()
-    })
+
+    await Promise.all(
+      Array.from(this.transportTable.values()).map((socket) => {
+        socket.removeEventListener('message', this.handleMessageEvent)
+        socket.close()
+      })
```

Comment:
I don't think there are any promises involved here?

Reply:
True. Was part of an experiment and left it by mistake 🙈